### PR TITLE
Add second email to send member their account details

### PR DESCRIPTION
**Pull request by mstratford**
Such that the original welcome email from SM is more personal and can be replied to.

Warning, this will require a config update to add the new email contents and update the existing welcome email.

### CODE REVIEW

**Comment by markspolakovs** [comment]
@mstratford code looks good on its face - can you rebase on top of current master so I can pull it in on dev?

**Comment by mstratford** [comment]
Will look at it soon

File: src/Classes/Config.php [hunk 1]
```diff
@@ -673,13 +673,24 @@ final class Config
...
+    /**** SIGNUP EMAILS ****/
+    public static $welcome_email_sender_memberid = null;
     public static $welcome_email = <<<EOT
 
-   This is a welcome email. You can use #NAME to get the user's first name,
-   and include #USER and #PASS to tell them their newly created login details.
+   This is a welcome email. You can use #NAME to get the user's first name.
+
+EOT;
+    public static $account_email = <<<EOT
```
Comment (LordAro) [inline src/Classes/Config.php]: this is very generically named, and the documentation below isn't much better. which account? which email? who gives the email to whom?

Comment (mstratford) [reply src/Classes/Config.php]: Better? :)

**Pull request merged**